bamboozle_unit_test.go

```diff
@@ -525,9 +526,7 @@ func runCheckCFCheckptSanityTestCase(t *testing.T, testCase *cfCheckptTestCase)
 	}
 	defer os.RemoveAll(tempDir)
 
-	db, err := walletdb.Create(
-		"bdb", tempDir+"/weks.db", true, dbOpenTimeout,
-	)
+	db, err := walletdb.Create("tempdb", "weks.db")
```

Comment:
Perhaps we can instead modify the tests to use a build tag to specify `tempdb` vs `bdb`?

Reply:
We could have a `NewTestDB` func defined for the build w/ and w/o the build tag defined.